tests/test/subTransactionRelayer.test.ts

```diff
@@ -99,6 +120,13 @@ describe('Substrate Transaction Relayer', function () {
       relayer: withdrawalProof.relayer,
     });
     expect(txHash).to.be.not.null;
+
+    // get the balance after withdrawal is done and see if it increases
+    // @ts-ignore
+    const { nonce: nonceAfter, data: balanceAfter } = await api.query.system.account(charlie.address);
+    let balanceAfterWithdraw = balanceAfter.free.toBigInt();
+    console.log(`balance after withdrawal is ${balanceAfter.free.toBigInt()}`);
+    expect(balanceAfterWithdraw > initialBalance).true;
```

Comment:
Try this instead, it is better in terms when the case fails it will print both values for you:
```suggestion
    expect(balanceAfterWithdraw).to.be.gt(initialBalance);
```

Reply:
Alright. Thanks

Reply:
Hello @shekohex i had to change it back to this `expect(balanceAfterWithdraw > initialBalance).true;`
because the test fails, `balanceAfterWithdraw` and ` initialBalance` are both BigInt and not number or date, therefore `expect(balanceAfterWithdraw).to.be.gt(initialBalance);` fails to assert